internal/handlers/pg/pgdb/databases.go

```diff
@@ -89,25 +90,27 @@ func CreateDatabaseIfNotExists(ctx context.Context, tx pgx.Tx, db string) error
 }
 
 // DropDatabase drops FerretDB database.
-//
-// It returns ErrSchemaNotExist if schema does not exist.
-func DropDatabase(ctx context.Context, tx pgx.Tx, db string) error {
-	_, err := tx.Exec(ctx, `DROP SCHEMA `+pgx.Identifier{db}.Sanitize()+` CASCADE`)
-	if err == nil {
-		return nil
+func DropDatabase(ctx context.Context, tx pgx.Tx, db string) (err error) {
+	defer contract.EnsureError(err, ErrSchemaNotExist) // if schema does not exist
```

Comment:
This approach looks helpful! We can also remove `case errors.Is(err, pgdb.ErrSchemaNotExist):` from `MsgDropDatabase`.

Reply:
We can't do that because contracts are not enforced in non-debug builds

Reply:
Ah, ok! I misunderstood the "It does nothing in non-debug builds" part. The part I didn't get is that the original error is kept.